src/WebServer.Logging/WebServer.Logging.csproj

```diff
@@ -0,0 +1,134 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why the extra project? If we want to implement another framework feature, say security, will we add another project "Webserver.Security"? I do understand the importance of separating the interfaces from the concretes, but we already have the Models project, why not merge those?